modules/misc/qt.nix

```diff
@@ -79,6 +79,18 @@ in {
         '';
       };
 
+      pluginPath = mkOption {
```

Comment:
Not a fan of this, imo we should do exactly what the nixos module does here: if the qt module is enabled, then set QT_PLUGIN_PATH and QML2_IMPORT_PATH to the user profile. There is no need for an option, and no need to make it customizable.

Reply:
Does it bring some side effect? It's not useless. We are occupying a environment variable without letting user to decide.

Reply:
Hmm, it's true that there's no way to extend this in home-manager like there is in nixos.
Let's still add the other variable.

Reply:
It's done 😉